README.md

```diff
@@ -1,5 +1,48 @@
 # Open vSwitch CNI plugin
 
+This plugin allows user to define Kubernetes networks on top of Open vSwitch bridges available on nodes. IPAM is currently not supported. There is no scheduling involved, desired bridges must be precreated on all nodes. Also, ovs-cni does not configure bridges, it's up to user to connect them to L2, L3 or an overlay network. Finally please note that Open vSwitch must be installed and running on the host.
+
+## Usage
+
+First define network definition. This object specifies to which Open vSwitch bridge should be pod attached and what VLAN ID should be set on the port. For more information, check [plugin documentation](docs/cni-plugin.md).
+
+```shell
+cat <<EOF | kubectl create -f -
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: ovs-conf
+spec:
+  config: '{
+      "cniVersion": "0.3.1",
+      "type": "ovs",
+      "bridge": "br1",
+      "vlan": 100
```

Comment:
is vlan optional? can we define a port without specifying a vlan?

Reply:
It's described in "For more information, check [plugin documentation](docs/cni-plugin.md)." Should I state it also here? I don't want to get into more and more details on from README.md as we add other paramters.